tests/sentry/nodestore/test_common.py

```diff
@@ -2,28 +2,36 @@
 Testsuite of backend-independent nodestore tests. Add your backend to the
 `ns` fixture to have it tested.
 """
-
+import pytest
+from contextlib import contextmanager
 from sentry.nodestore.django.backend import DjangoNodeStorage
-from sentry.nodestore.bigtable.backend import BigtableNodeStorage
-from tests.sentry.nodestore.bigtable.backend.tests import MockedBigtableNodeStorage
+from tests.sentry.nodestore.bigtable.backend.tests import (
+    MockedBigtableNodeStorage,
+    get_temporary_bigtable_nodestorage,
+)
 
-import pytest
+
+@contextmanager
+def nullcontext(returning):
+    # TODO: Replace with ``contextlib.nullcontext`` after upgrading to 3.7
+    yield returning
 
 
 @pytest.fixture(
     params=["bigtable-mocked", "bigtable-real", pytest.param("django", marks=pytest.mark.django_db)]
 )
 def ns(request):
-    if request.param == "bigtable-real":
-        pytest.skip("Bigtable is not available in CI")
-
-    ns = {
-        "bigtable-mocked": lambda: MockedBigtableNodeStorage(project="test"),
-        "bigtable-real": lambda: BigtableNodeStorage(project="test"),
-        "django": lambda: DjangoNodeStorage(),
-    }[request.param]()
-    ns.bootstrap()
-    return ns
+    # backends are returned from context managers to support teardown when required
```

Comment:
Alternate approach would be to add `drop` or `destroy` or something to `NodeStorage` interface that provides a consistent point for a request finalizer. No strong feelings one way or the other on which approach is best.